### ISSUE TITLE
qptiff format

### ISSUE THREAD

**Issue by alexeykb** [BodenmillerGroup/steinbock#236]
Hello!

I was wondering if it is possible to use qptiff as an input format for steinbock ?  If not, what is the preferable way to transform it to appropriate tiff?

Thank you
Best regards
Alexey

**Comment by Milad4849** [comment]
@alexeykb Thank you for reaching out! External images need to be in a format supported by [imageio](https://imageio.readthedocs.io/) (see External images in the [relevant part of the documentation](https://bodenmillergroup.github.io/steinbock/latest/cli/preprocessing/)). I am not aware of the best practices for qptiff conversion. You would need to convert them into a iamgeio supported format and then convert them to steinbock-compatible TIFF files using `steinbock preprocess external images`.

**Comment by Milad4849** [comment]
Closing this due to inactivity

**Comment by alexeykb** [comment]
Hello, I have an update on current issue (still doesnt work):
I am facing the issue with segmenting the ome.tiff using steinbock.
I have original CODEX files in the format of qptiff. I have transformed qptiff to ome.tiff using  BioFormats:
**https://docs.openmicroscopy.org/bio-formats/6.10.1/users/comlinetools/index.html**
What increased my original file from 25 gb to 75gb and stored it to:
**path/to/publication/steinbock/external**

I further have tried to input it to the IMC pipline (starting with steinbock), by using this command:

**steinbock preprocess external images --img /data/external --imgout /data/images --infoout /data/images/images.csv**

Unfortunatelly , it doesnt work and return the error:
**WARNING steinbock.preprocessing.external - Unsupported file format: /data/external/scanone.ome.tiff**

However, originally, the channel names from the ome.tiff 
I have extracted the channel names from the ome.tiff file and those are in the form
Channel:0:0
Channel:0:1 etc

Which is not acceptable for steinbock, so i changed it to the 
0
1 etc



What should be my steps now ?  Should I open ome.tiff through qptiff and manually rename channels  or the problem is in ome.tiff file, and not in channels names. If it is ome.tiff fault, how can I overpass it ? I also tried to transform qptiff thorugh QPath and it didi not give any positive results either.

Thank you
Best
Alexey
